db/migrations/2022_05_23_000000_increase_tshift_field_width.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Engelsystem\Migrations;
+
+use Engelsystem\Database\Migration\Migration;
+
+/**
+ * To allow for larger key names such as "2XL-G"
+ */
+class IncreaseTshiftFieldWidth extends Migration
+{
+    /**
+     * Run the migration
+     */
+    public function up()
+    {
+        if (!$this->schema->hasTable('GroupPrivileges')) {
+            return;
+        }
+
+        $connection = $this->schema->getConnection();
+
+        $connection->statement(
+            'ALTER TABLE users_personal_data MODIFY shirt_size VARCHAR(6) ;'
```

Comment:
To be portable, the creation of the statement should be done by the db abstraction and not manually as that forces it to be only MySQL compatible and breaks for SQLite which is used for tests.
A previous migration is for example [ChangeUsersContactDectFieldSize](https://github.com/engelsystem/engelsystem/blob/main/db/migrations/2018_12_21_000000_change_users_contact_dect_field_size.php#L43) which uses
```php
$table->string($column, $length)->change();
```

Also i think at least 10 would be eventually better here to have more room for others?